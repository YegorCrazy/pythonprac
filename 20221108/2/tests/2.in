('a', 'b'), ('c', 'd'), ('f', 'g')
(1, 3), (4, 1), (10, 2), (6, 8)
(0, 0), (0, 0), (0, 0)

