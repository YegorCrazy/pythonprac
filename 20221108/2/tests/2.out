Not a triangle
Invalid input
Not a triangle
